config/services/RH-Satellite-6.xml

```diff
@@ -3,10 +3,9 @@
   <short>Red Hat Satellite 6</short>
   <description>Red Hat Satellite 6 is a systems management server that can be used to configure new systems, subscribe to updates, and maintain installations in distributed environments.</description>
   <include service="foreman"/>
-  <port protocol="tcp" port="5000"/>
+  <include service="mqtt"/>
+  <include service="cockpit"/>
```

Comment:
While this is using the same port as cockpit, cockpit itself isn't used here. Would this be confusing down the road?

Reply:
100% agree. Should only include `cockpit` if it's actually using cockpit.

Reply:
Thank you for your comment @jcpunk !

I reverted this with ff823fa